backend/src/services/auto_trader.py

```
@@ -33,238 +31,203 @@ def __init__(self, socketio=None, data_fetcher_service=None, ai_analysis_service
 
     def _load_config(self) -> Dict:
         if os.path.exists(CONFIG_FILE):
             try:
                 with open(CONFIG_FILE, 'r') as f:
-                    config = json.load(f)
-                    logger.info(f"Loaded auto-trader config from {CONFIG_FILE}")
-                    return config
-            except Exception as e:
-                logger.error(f"Error loading config from {CONFIG_FILE}: {e}. Using default config.")
+                    return json.load(f)
+            except Exception: pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)